### DISCUSSION TITLE
Error with removing intermediary files after updating to 1.3

### DISCUSSION THREAD

**Discussion by waurimal** [quarto-dev/quarto-cli#5430] [Q&A]
You have a fresh installation of quarto 1.3.340.
When I try to publish an existing project that I am creating in the previous version, I get the following error. It published fine in the previous version.
Why am I getting this error after installing the new version?

-------------
Rendering for publish:

[ 1/14] index.qmd
ERROR: NotFound: 지정된 파일을 찾을 수 없습니다. (os error 2), remove 'G:\My drive\2023\00_SNUE\01\AAA\main\HEMP\index_files'

NotFound: 지정된 파일을 찾을 수 없습니다. (os error 2), remove 'G:\My drive\2023\00_SNUE\01\AAA\main\HEMP\index_files'
    at Object.removeSync (deno:runtime/js/30_fs.js:157:9)
    at removeIfEmptyDir (file:///C:/Program%20Files/Quarto/bin/quarto.js:5701:18)
    at renderCleanup (file:///C:/Program%20Files/Quarto/bin/quarto.js:80573:5)    at cleanupExecutedFile (file:///C:/Program%20Files/Quarto/bin/quarto.js:99089:5)
    at file:///C:/Program%20Files/Quarto/bin/quarto.js:98809:17
    at Array.forEach (<anonymous>)
    at cleanupExecutedFiles (file:///C:/Program%20Files/Quarto/bin/quarto.js:98808:35)
    at Object.onComplete (file:///C:/Program%20Files/Quarto/bin/quarto.js:98887:17)
    at renderFiles (file:///C:/Program%20Files/Quarto/bin/quarto.js:86652:42)
    at async renderProject (file:///C:/Program%20Files/Quarto/bin/quarto.js:86794:25)

**Comment by mcanouil** [comment]
Thanks for reporting this possible bug, could you share a repository and/or a small reproducible example to use to investigate?

**Reply by waurimal** [reply]
It seems to be a Google Drive issue.
I copied the same content to my hard disk and ran it.
I created a new project in Google Drive and ran the basic example and got the same error.

**Reply by mcanouil** [reply]
Thanks for figuring out that part.

There are several opened issues about using network/cloud volumes, see <https://github.com/quarto-dev/quarto-cli/issues?q=is%3Aopen+is%3Aissue+label%3Afile-systems>.

**Reply by waurimal** [reply]
Thanks for your interest. It was helpful.

**Reply by cderv** [reply]
I think this is related to that one: https://github.com/quarto-dev/quarto-cli/issues/4614 - I'll reinvestigate. Possilby linked to Deno and network drive. 

> I created a new project in Google Drive and ran the basic example and got the same error.

What is the basic example your tried ? Just to use the same ?

**Reply by cderv** [reply]
I can confirm this is the same as #4614 but error message is different, probably because of `embed-resources`

````markdown
---
title: test
embed-resources: true
---

```{r}
plot(1 + 1)
```
````

I can repo with this
